### DISCUSSION TITLE
How do I add a new Exchange support?

### DISCUSSION THREAD

**Discussion by SeamonYao** [nautechsystems/nautilus_trader#1643] [Ideas]
I want to support a new Exchange to nautilus_trader, What steps should I follow? Thanks.

**Comment by cjdsellers** [comment]
Hi @SeamonYao

Thanks for your interest.

We actually have an initial develop guide for this [here](https://docs.nautilustrader.io/nightly/developer_guide/adapters.html).

Hope that helps!

**Reply by fredmonroe** [reply]
that link seems broken for me. maybe the nightly docs are busted at this moment.

i see information here:
https://docs.nautilustrader.io/developer_guide/adapters.html

@SeamonYao - look at that link in the Execution Client section, it might help you out

**Reply by cjdsellers** [reply]
Thanks, the nightly docs are now repaired - this happens following a deployment until another push to `nightly`, just a small devops thing which needs to be tidied up.
